Makefile

```diff
@@ -1,4 +1,5 @@
 JAVAC ?= javac
+GIT ?= git
```

Comment:
There's no real use to git being a variable (the chances of us wanting e.g. 2 versions of it are vanishingly small), so let's simplify and remove it.